diff -uNr dc-qt-0.1.1.orig/src/player_xine.cc dc-qt-0.1.1/src/player_xine.cc
--- dc-qt-0.1.1.orig/src/player_xine.cc	2004-03-11 18:22:43.000000000 +0100
+++ dc-qt-0.1.1/src/player_xine.cc	2004-07-28 13:09:04.490690240 +0200
@@ -1063,7 +1063,7 @@
 
 // Define a bunch of dummy methods
 
-av_player_dummy::av_player_xine(QWidget *parent, char *name) : 
+av_player_xine::av_player_xine(QWidget *parent, char *name) : 
   av_player(parent, name)
 {
   // Set black widget backgound
@@ -1076,12 +1076,14 @@
   }
 }
 
-av_player_dummy::~av_player_xine() { disconnect(); }
+av_player_xine::~av_player_xine() { disconnect(); }
 bool av_player_xine::init() { return true; }
 void av_player_xine::shutdown() { }
-bool av_player_xine::play(QString &) { emit sig_playing(); return true; }
+bool av_player_xine::play(QString &, QString &) { emit sig_playing(); return true; }
 bool av_player_xine::stop() { /*emit sig_stopped();*/ return true; }
 bool av_player_xine::pause() { return true; }
+bool av_player_xine::seek(int) { return true; }
+int av_player_xine::get_progress() { return 0; }
 bool av_player_xine::ffwd() { return true; }
 bool av_player_xine::rwnd() { return true; }
 bool av_player_xine::set_fullscreen(bool) { return false; }
@@ -1090,7 +1092,7 @@
 bool av_player_xine::playing_audio() { return false; }
 bool av_player_xine::playing_video() { return false; }
 bool av_player_xine::playing_fullscreen() { return false; }
-av_player_status av_player_xine::get_status() { return 0; }
+av_player::av_player_status av_player_xine::get_status() { return DISABLED; }
 
 // endif DISABLE_PREVIEW
 #endif
diff -uNr dc-qt-0.1.1.orig/src/player_xine.h dc-qt-0.1.1/src/player_xine.h
--- dc-qt-0.1.1.orig/src/player_xine.h	2004-03-11 18:22:43.000000000 +0100
+++ dc-qt-0.1.1/src/player_xine.h	2004-07-28 13:06:24.197058576 +0200
@@ -37,6 +37,7 @@
 class av_player_xine : public av_player {
 
   Q_OBJECT
+  void slot_emit_signals();
 
 public:
 
@@ -165,9 +166,6 @@
   xine_audio_port_t   *ao_port;
   xine_event_queue_t  *event_queue;
 
-private slots:
-  void slot_emit_signals();
-
 // endif DISABLE_PREVIEW
 #endif
 
